symphony/lib/toolkit/class.entrymanager.php

```diff
@@ -408,13 +412,15 @@ public static function fetch($entry_id = null, $section_id = null, $limit = null
         // Handle when the sort field is an actual Field
         } elseif (self::$_fetchSortField && $field = FieldManager::fetch(self::$_fetchSortField)) {
             if ($field->isSortable()) {
-                $field->buildSortingSQL($joins, $where, $sort, self::$_fetchSortDirection, $select);
+                $field->buildSortingSQL($joins, $where, $sort, self::$_fetchSortDirection);
+                $sortSelectClause = $field->buildSortingSelectSQL($sort, self::$_fetchSortDirection);
```

Comment:
Well I've modeled the name according to the existing analogous method `buildSortingSQL`. We have to stick with the current nomenclature, even if it's not the best one IMHO.